File: web-development/helsinki-mooc/java/ex16.java
import java.util.Scanner;

public class ex16 {
    public static void main(String[] args) {
      Scanner input = new Scanner(System.in);

      System.out.print("Type a number: ");
      int num = Integer.parseInt(input.nextLine());

      if (num % 2 == 0) {
        System.out.println(num + " is even!");
      } else if (num % 2 != 0) {
        System.out.println(num + " is odd!");
      } else {
        System.out.println("Ooooops! something went wrong.");
      }

    }// main entry func
}

File: compsci/programming/helsinki-moocs/c/ex16.c
#include <stdio.h>

int main(void) {
  int x;

  printf("Type a number: ");
  scanf("%d", &x);

  if (x >= 0) {
    printf("The number %d is positive\n", x);
  }
  else {
    printf("The number %d is negative\n", x);
  }

  if (x % 2 == 0) {
    printf("And the number is EVEN\n");
  }
  else {
    printf("And the number is ODD\n");
  }

  return 0;
}
